cms/envs/common.py

```diff
@@ -868,7 +868,8 @@
     'django_nose',
 
     # For CMS
-    'contentstore',
+    'contentstore.apps.ContentstoreConfig',
```

Comment:
This is the only place I have a question, and it's just for my own understanding. What was the line that got deleted doing?